bin/fillTestData/Users.js

```diff
@@ -116,6 +116,7 @@ var addUsers = function (max, adminUsers) {
           user.public = true;
           user.avatarUploaded = false;
           user.avatarSource = 'none';
+          user.welcomeSequenceStep = 3;
```

Comment:
I am assuming that is the last step in welcome emails, but I couldn't find elsewhere in the code this value getting set to 3. On that note when we switch to ES6, it would be helpful to use consts so we can enumerate what each step is.

Reply:
Yep it's the last one; It's set in this job: https://github.com/Trustroots/trustroots/blob/15a8bcebee8286eca1591f950a120420b2e08dd3/modules/users/server/jobs/user-welcome-sequence-third.server.job.js

Using `$inc` https://docs.mongodb.com/manual/reference/operator/update/inc/

Thanks for looking into it!